identify/extensions.py

```diff
@@ -288,6 +288,8 @@
     'Gemfile': EXTENSIONS['rb'],
     'Gemfile.lock': {'text'},
     'GNUmakefile': EXTENSIONS['mk'],
+    'go.mod': EXTENSIONS['go'],
+    'go.sum': EXTENSIONS['go'],
```

Comment:
I think these probably would get classified a little differently since they aren't actually "go" go source, but a different related format?

Reply:
That is correct, they are not go code. Do you have a recommendation for this? Perhaps we can call these something unique such as go-mod and go-sum?

Reply:
yeah `{'text', 'go-mod'}` and `{'text', 'go-sum}` probably makes sense -- that way if a tool specifically targets those it can differentiate

Reply:
should be good to go now.